js/packages/web/src/pages/index.tsx

```diff
@@ -13,6 +15,14 @@ interface AppProps {
   storefront: Storefront;
 }
 
+Bugsnag.start({
+  apiKey: 'bd22af1f4e1213b5badb590a48128bf1',
```

Comment:
nb: can set as an environment variable and expose with NEXT_PUBLIC_BUGSNAG_API_KEY